base/applications/rapps/include/rosui.h

```diff
@@ -302,54 +300,78 @@ class CUiBox
         *newRect = currentRect;
     }
 
-
 public:
+    //virtual ~CUiBox() {} // Needed for current CUiCollection implementation.
```

Comment:
This is a leftover of tests back when I made `CUiBox` derive from `CUiCollection` in order to simplify code later on (regarding the `AsBox()` returned values checks).